ui/raidboss/data/05-shb/raid/e8n.js

```diff
@@ -6,8 +6,520 @@
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DE3', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DE3', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DE3', capture: false }),
+      condition: function(data) {
+        return data.mirrorsActive;
+      },
+      infoText: {
+        en: 'Get in, then close to a mirror',
+      },
+    },
+    {
+      id: 'E8N Mirror Cleanup',
+      regex: Regexes.startsUsing({ source: 'Frozen Mirror', id: ['4DFE', '4DFF', '4E00', '4E01'], capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Eisspeigel', id: ['4DFE', '4DFF', '4E00', '4E01'], capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'miroir de glace', id: ['4DFE', '4DFF', '4E00', '4E01'], capture: false }),
+      regexJa: Regexes.startsUsing({ source: '氷面鏡', id: ['4DFE', '4DFF', '4E00', '4E01'], capture: false }),
+      // FIXME: CN/KO translations.
+      // Maybe not necessary to delay here, but just to be safe.
+      delaySeconds: 5,
+      run: function(data) {
+        data.mirrorsActive = false;
+      },
+    },
+    {
+      id: 'E8N Absolute Zero',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DD7', capture: false }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DD7', capture: false }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DD7', capture: false }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DD7', capture: false }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DD7', capture: false }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DD7', capture: false }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E8N Double Slap',
+      regex: Regexes.startsUsing({ source: 'Shiva', id: '4DDA' }),
+      regexDe: Regexes.startsUsing({ source: 'Shiva', id: '4DDA' }),
+      regexFr: Regexes.startsUsing({ source: 'Shiva', id: '4DDA' }),
+      regexJa: Regexes.startsUsing({ source: 'シヴァ', id: '4DDA' }),
+      regexCn: Regexes.startsUsing({ source: '希瓦', id: '4DDA' }),
+      regexKo: Regexes.startsUsing({ source: '시바', id: '4DDA' }),
+      condition: Conditions.caresAboutPhysical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'E8N Frigid Water',
+      regex: Regexes.headMarker({ id: '0057' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Flare on YOU',
+        de: 'Flare auf DIR',
+      },
+    },
+    {
+      id: 'E8N Icicle Impact',
+      regex: Regexes.headMarker({ id: '0060' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'E8N Puddle Chase',
+      regex: Regexes.headMarker({ id: '00C5' }),
+      condition: Conditions.targetIsYou(),
+      alertText: {
+        en: 'Dropping puddles--Stay out',
+      },
+    },
+    {
+
+      id: 'E8N Stoneskin',
```

Comment:
I haven't added this to savage yet because I wanted to add something to figure out which side the player was on before alerting to the other side's stoneskin.  Is the order of the adds the same on both sides for normal mode?

Reply:
It appears to be, so far as I can tell, but I'm fine removing that for now given that it's uncertain. It's a minor annoyance at best.